docs/tutorials/10-minute-walkthrough/index.md

```diff
@@ -107,18 +82,13 @@ Same as the previous step, but we'll output to ElasticSearch now.
 
 ### Run it
 
-    bin/logstash agent -f apache-elasticsearch.conf web
+    bin/logstash agent -f apache-elasticsearch.conf
```

Comment:
yeah that caught my attention, so..

Lets keep this change only for ripping out web and open another PR for these things